lib/src/sdk_definitions.dart

```diff
@@ -825,6 +825,21 @@ void _addToMapIfNotNull(String key, dynamic value, Map<String, dynamic> map) {
   }
 }
 
+class ExternalNavigation {
+  String messageType;
+  Map<String, dynamic> payload;
+
+  ExternalNavigation(this.messageType, this.payload);
+}
+
+class ExternalNavigationMessageType {
+  static const navigationStarted = "NavigationStarted";
+  static const navigationUpdated = "NavigationUpdated";
+  static const destinationReached = "DestinationReached";
+  static const outsideRoute = "OutsideRoute";
```

Comment:
Por outra parte "outsideRoute" nunca o usamos, sempre se usa "out of route", por que este cambio?

Reply:
Estos nomes xa se subiron a produccion en Cordova e no SDK. Ao final estos strings indican o tipo de mensaxe de navegacion aos SDKs, e estes internamente deciden que facer. En caso de cambialos haberia que tocar SDKs + Cordova.